lms/djangoapps/learner_recommendations/tests/test_serializers.py

```diff
@@ -81,3 +83,74 @@ def test_happy_path(self):
                 "isControl": False,
             },
         )
+
+
+class TestCrossProductRecommendationsSerializer(TestCase):
+    """Tests for the Cross Product Recommendations Serializer"""
+
+    def mock_recommended_courses(self, num_of_courses):
+        """Course data mock"""
+
+        recommended_courses = []
+
+        for i in range(num_of_courses):
+            recommended_courses.append(
+                {
+                    "key": f"edx+HL{i}",
+                    "uuid": f"{i}f8cb2c9-589b-4d1e-88c1-b01a02db3a9c",
+                    "title": f"Title {i}",
+                    "image": {
+                        "src": f"https://www.logo_image_url{i}.com",
+                    },
+                    "url_slug": f"https://www.marketing_url{i}.com",
+                    "course_type": "executive-education",
+                    "owners": [
+                        {
+                            "key": f"org-{i}",
+                            "name": f"org {i}",
+                            "logo_image_url": f"https://discovery.com/organization/logos/org-{i}.png",
+                        },
+                    ],
+                    "course_runs": [
+                        {
+                            "key": f"course-v1:Test+2023_T{i}",
+                            "marketing_url": f"https://www.marketing_url{i}.com",
+                            "availability": "Current",
+                        }
+                    ],
+                    "active_course_run": {
+                        "key": f"course-v1:Test+2023_T{i}",
+                        "marketing_url": f"https://www.marketing_url{i}.com",
+                        "availability": "Current",
+                    },
+                    "location_restriction": None
+                },
+            )
+
+        return recommended_courses
+
+    def test_successful_serialization(self):
+        courses = self.mock_recommended_courses(num_of_courses=2)
+
+        serialized_data = CrossProductRecommendationsSerializer({
+            "courses": courses
+        }).data
+
+        self.assertDictEqual(
+            serialized_data,
+            mock_course_data
+        )
+
+    def test_no_course_data_serialization(self):
+        courses = self.mock_recommended_courses(num_of_courses=0)
+
+        serialized_data = CrossProductRecommendationsSerializer({
+            "courses": courses
+        }).data
```

Comment:
nit:
```suggestion
        serialized_data = CrossProductRecommendationsSerializer({
            "courses": [],
        }).data
```

Reply:
Makes much more sense, this one was silly from my side 😅